lib/rubocop/cop/rspec/describe_class.rb

```diff
@@ -24,7 +25,7 @@ class DescribeClass < Cop
         FEATURE_PAIR = s(:pair, s(:sym, :type), s(:sym, :feature))
 
         MESSAGE = 'The first argument to describe should be the class or ' \
-                  'module being tested.'
+                  'module being tested.'.freeze
```

Comment:
I may have misunderstood it, but doesn't setting `frozen_string_literal` mean that there's no need to call `.freeze` on each string?